README.md

```diff
@@ -69,6 +69,7 @@ Contributions are most welcome, please adhere to the [contribution guidelines](c
 - [GPT-NeoX](https://github.com/EleutherAI/gpt-neox) ![](https://img.shields.io/github/stars/EleutherAI/gpt-neox.svg?style=social) - An implementation of model parallel autoregressive transformers on GPUs, based on the DeepSpeed library.
 - [Luotuo](https://github.com/LC1332/Luotuo-Chinese-LLM) ![](https://img.shields.io/github/stars/LC1332/Luotuo-Chinese-LLM.svg?style=social) - A Chinese LLM, Based on LLaMA and fine tune by Stanford Alpaca, Alpaca LoRA, Japanese-Alpaca-LoRA.
 - [Mixtral-8x7B-v0.1](https://huggingface.co/mistralai/Mixtral-8x7B-v0.1) - The Mixtral-8x7B Large Language Model (LLM) is a pretrained generative Sparse Mixture of Experts.
+- [Prompteams] (https://www.prompteams.com/) - Prompt management system. Version, test, collaborate, and retrieve prompts through real-time API's. Have GitHub style with repos, branches, and commits (and commit history).
```

Comment:
```suggestion
- [Prompteams] (https://www.prompteams.com/) - Prompt management system. Version, test, collaborate, and retrieve prompts through real-time APIs. Have GitHub style with repos, branches, and commits (and commit history).
```

Also I think a better place for this would be inside the `LLMOps` section. Can you move it?

Reply:
Moved @strickvl . Agreed. Read the wrong heading